Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM runtimeverificationinc/kframework-k:ubuntu-focal-${K_COMMIT}
 RUN    apt-get update            \
     && apt-get upgrade --yes     \
     && apt-get install --yes     \
+            bison                \
```

Comment:
The focal package for bison is only version 3.5.1, which will fail the version check.

In order to get CI to use the bison generated parser, you'll need to add a RUN instruction to the dockerfile which builds and installs a more recent version of bison.